mlrun/feature_store/api.py

```diff
@@ -311,6 +315,10 @@ def ingest(
 
     if mlrun_context:
         # extract ingestion parameters from mlrun context
+        if isinstance(source, pd.DataFrame):
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "DataFrame source is illegal when running ingest with mlrun context"
```

Comment:
"with mlrun context" won't mean anything to a user, explain it in their words

Reply:
mlrun context is a parameter that the user passes to ingest function